libnetdata/threads/threads.c

```diff
@@ -134,6 +134,25 @@ static void thread_set_name(NETDATA_THREAD *nt) {
     }
 }
 
+void uv_thread_set_name(uv_thread_t ut, const char* name) {
+    int ret=0;
+
+    // Name is limited to 16 chars
+    char threadname[16];
```

Comment:
It's a good idea to use a `#define` here since we also have the function `thread_set_name()` above.

Reply:
Done.
`#define NETDATA_THREAD_NAME_MAX 15`